CONTRIBUTING.md

```diff
@@ -31,45 +38,56 @@ Ready to contribute? Here's how to set up `simulariumio` for local development.
     Now you can make your changes locally.
 
 5. When you're done making changes, check that your changes pass linting and
-   tests, including testing other Python versions with make:
+   tests with [just](https://github.com/casey/just):
 
     ```bash
-    make build
+    just build
     ```
 
 6. Commit your changes and push your branch to GitHub:
 
     ```bash
     git add .
-    git commit -m "Resolves gh-###. Your detailed description of your changes."
+    git commit -m "Your detailed description of your changes."
     git push origin {your_development_type}/short-description
     ```
 
 7. Submit a pull request through the GitHub website.
 
-___
+## Just Commands
 
-## Deploying
-
-A reminder for the maintainers on how to deploy.
-Make sure all your changes are committed.
-Then run:
+For development commands we use [just](https://github.com/casey/just).
 
 ```bash
-$ bumpversion patch # possible: major / minor / patch
-$ git push
-$ git push --tags
-git branch -D stable
-git checkout -b stable
-git push --set-upstream origin stable -f
+just
+```
+```
+Available recipes:
+    build                    # run lint and then run tests
+    clean                    # clean all build, python, and lint files
+    default                  # list all available commands
+    generate-docs            # generate Sphinx HTML documentation
+    install                  # install with all deps
+    lint                     # lint, format, and check all files
+    release                  # release a new version
+    serve-docs               # generate Sphinx HTML documentation and serve to browser
+    tag-for-release version  # tag a new version
+    test                     # run tests
+    update-from-cookiecutter # update this repo using latest cookiecutter-py-package
 ```
 
-This will release a new package version on Git + GitHub and publish to PyPI.
-
-___
+## Deploying
 
-## Questions or Thoughts?
+A reminder for the maintainers on how to deploy.
+Make sure the main branch is checked out and all desired changes
+are merged. Then run:
 
-Talk to us on [one of our community forums][community].
+```bash
+just tag-for-release "vX.Y.Z"
+just release
+```
```

Comment:
Versions are entirely controlled by git tags! It means that even when you pip install editable mode (`just install`) it will have like a `3.1.2.dev{git_hash}`!